sample-3.md

```diff
@@ -0,0 +1,537 @@
+# Step-by-step: Deploy "Hello World" Eventing soluction
+
+This step-by-step guide will guide you through all the steps involved to 
+run “Hello World” eventing solution, where you will send an event to the 
+Knative eventing broker inside your K8s namespace and receive it in a pod 
+using Knative eventing. ETA - 20-30 mins. If you want to skip all the steps 
+and want to quickly deploy the solution then please refer to 
+[Quick Start: Deploy "Hello World" eventing solution](todo).
+
+## Before you begin
+
+Before you begin:
+- Have a Kubernetes Cluster running version 1.12.
```

Comment:
This would depend on the version of eventing and will be difficult to maintain. How about just pointing to "isntall knative eventing", which will take care of K8s pre-req